wasp/wasp.py

```diff
@@ -205,6 +207,24 @@ def notify_duration(self):
         """Cached copy of the current vibrator pulse duration in milliseconds"""
         return self._nfylev_ms
 
+    @property
+    def vibe_on_notif(self):
+        """Vibrate when a notification is received bool"""
+        return self._vibe_on_notif
+
+    @vibe_on_notif.setter
+    def vibe_on_notif(self, value):
+        self._vibe_on_notif = value
+
+    @property
+    def wake_on_notif(self):
+        """Wake the screen when a notification is received bool"""
+        return self._wake_on_notif
+
+    @wake_on_notif.setter
+    def wake_on_notif(self, value):
+        self._wake_on_notif = value
+
```

Comment:
There's no need for these to use properties... just a regular member variable (without the leading underscore) is fine.

Reply:
> Interesting idea. Needs a couple of tweaks but I think it could be useful (at least until we get lift-to-wake working with BMA421).

PS Can you take a look at the DCO failure as well (the Signed-off-by: needs to be in the commit rather than the PR). I think clicking on the Details next to the failure will give some hints on how to do this in git.